ledger-tool/src/main.rs

```diff
@@ -910,6 +910,12 @@ fn main() {
         .validator(is_parsable::<usize>)
         .takes_value(true)
         .help("How many accounts to add to stress the system. Accounts are ignored in operations related to correctness.");
+    let accounts_filler_size = Arg::with_name("accounts_filler_size")
```

Comment:
`clap::Arg::requires("accounts_filler_count")` since the new arg is only relevant with filler accounts enabled